src/schema/article-renderer.tsx

```diff
@@ -40,6 +40,8 @@ import type { GeogebraProps } from '@/components/content/geogebra'
 import type { InjectionProps } from '@/components/content/injection'
 import type { MathProps } from '@/components/content/math'
 import type { VideoProps } from '@/components/content/video'
+// eslint-disable-next-line import/extensions
+import { EditorState } from '@/pages/[...slug]'
```

Comment:
@inyono Looks like a bug in the ESlint config.

Reply:
Full error:
`Unexpected use of file extension "slug]" for "@/pages/[...slug]"eslintimport/extensions`

Reply:
that's why I asked about the `...slug` above